### ISSUE TITLE
WADI data

### ISSUE THREAD

**Issue by KennyNH** [elisejiuqizhang/USAD-on-WADI-and-SWaT#5]
Thanks for your implementation.

There are 1209601 rows in 'WADI_14days.csv'. But why, in the paper of USAD, Table 1 shows that the length of training data of WADI is 1048517?

**Comment by KennyNH** [comment]
Sorry to trouble you, but I have another question.

The time segments of attacke 3-4 and 9 seem to be overlapped. Is this an error of labels provided by WADI? As the attacks are listed in time order.
